usr/src/test/zfs-tests/cmd/scripts/zfstest.ksh

```diff
@@ -176,6 +176,8 @@ export KEEP="^$(echo $KEEP | sed 's/ /$|^/g')\$"
 num_disks=$(echo $DISKS | awk '{print NF}')
 [[ $num_disks -lt 3 ]] && fail "Not enough disks to run ZFS Test Suite"
 
+sudo -k coreadm -e process
```

Comment:
This effectively changes coreadm settings and doesn't restore them, which makes this change a bit pointless -- we could just mention the needed settings in README instead.

Reply:
i think will be better to save current state -> update it for tests -> restore it
no need just point it in README

Reply:
Or that, I just don't like system-wide settings being modified without me knowing about it.

BTW, the change itself is missing the ```-i core```, which was in the description.

Reply:
The `-i core` from the description was changed during review - the description is wrong.
I think it's better to have this command in the wrapper script than have a comment in the README that folks may or may not read. I also feel that it's appropriate to change system wide settings on a test system.
That said, if this is a blocker for you, let us know; it's easy enough to restore the setting. Alternatively, we could simply abandon this PR - I don't expect that coreadm settings need to be changed on other platforms.